api/events/MSAMEventCollector.yml

```diff
@@ -19,6 +19,7 @@ Resources:
             - Effect: Allow
               Action:
                 - dynamodb:PutItem
+                - medialive:Describe*
```

Comment:
Just to clarify, without this we get an unauthorized error when attempting `client.describe_channel` in the Lambda